aiohttp/pytest_plugin.py

```diff
@@ -230,6 +230,23 @@ def proactor_loop():  # type: ignore
         yield _loop
 
 
+@pytest.fixture
+def selector_loop():  # type: ignore
+    if not PY_37:
+        policy = asyncio.get_event_loop_policy()
+        policy._loop_factory = asyncio.SelectorEventLoop  # type: ignore
+    else:
+        if PY_38:
+            policy = asyncio.WindowsSelectorEventLoopPolicy()  # type: ignore
+        else:
+            policy = asyncio.WindowsProactorEventLoopPolicy()  # type: ignore
+        asyncio.set_event_loop_policy(policy)
+
+    with loop_context(policy.new_event_loop) as _loop:
+        asyncio.set_event_loop(_loop)
+        yield _loop
```

Comment:
Is it possible to unset the loop after this `yield`?

Reply:
I'm not sure it makes sense since fixture executed once for each test, so there's no need to unset the loop at all, but I can be wrong. Also, see [loop fixture](https://github.com/aio-libs/aiohttp/blob/master/aiohttp/helpers.py#L57) and [proactor_loop fixture](https://github.com/aio-libs/aiohttp/blob/master/aiohttp/pytest_plugin.py#L220)